packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1461,6 +1469,25 @@
   "Password": "Password",
   "Password_Change_Disabled": "Your Rocket.Chat administrator has disabled the changing of passwords",
   "Password_changed_successfully": "Password changed successfully",
+  "Password_Policy": "Password Policy",
+  "Accounts_Password_Policy_Enabled": "Enable Password Policy",
+  "Accounts_Password_Policy_Enabled_Description": "When enabled, user passwords must adhere to the policies set forth. Note: this only applies to new passwords, not existing passwords.",
+  "Accounts_Password_Policy_MinLength": "Minimum Length",
+  "Accounts_Password_Policy_MinLength_Description": "Ensures that passwords must have at least this amount of characters. Use `-1` to disable.",
+  "Accounts_Password_Policy_MaxLength": "Maximum Length",
+  "Accounts_Password_Policy_MaxLength_Description": "Ensures that passwords do not have more than this amount of characters. Use `-1` to disable.",
+  "Accounts_Password_Policy_ForbidRepeatingCharacters": "Forbid Repeating Characters",
+  "Accounts_Password_Policy_ForbidRepeatingCharacters_Description": "Ensures passwords do not contain the same character repeating next to each other.",
+  "Accounts_Password_Policy_ForbidRepeatingCharactersCount": "Max Repeating Characters",
+  "Accounts_Password_Policy_ForbidRepeatingCharactersCount_Description": "The amount of times a character can be repeating before it is not allowed.",
```

Comment:
I believe this description doesn't match the implementation. If I set the max character count to 3, should it accept 3 and block 4 or accept 2 and block 3?

Based on the description I would think it would be the first, but it was implemented the other way.

Reply:
Now it's working like the description